src/tools/publish_toolstate.py

```diff
@@ -22,6 +22,7 @@
     'nomicon': '@frewsxcv @Gankro',
     'reference': '@steveklabnik @Havvy @matthewjasper @alercah',
     'rust-by-example': '@steveklabnik @marioidival @projektir',
+    'embedded-book': '',
```

Comment:
cc @jamesmunns could you suggest a list of people to ping when embedded-book's test status changed?

Reply:
Sure! I just added that as a suggestion. This is the [Resources Team](https://github.com/rust-embedded/wg#the-resources-team) in the `rust-embedded` organization, but I don't think you can cross tag organizations, like @rust-embedded/Resources, unfortunately.